css/common.css

```diff
@@ -98,6 +98,10 @@ summary > .oo-ui-labelElement-label {
 
 /* Wiki list */
 
+.oo-ui-messageWidget.oo-ui-messageWidget-block.oo-ui-flaggedElement-warning {
+	margin-top: 16px;
```

Comment:
Use 1em for the margin

Reply:
I found that .showClosed had -6px for margin-top.
In index.php I found that this element is for the warning box. I've removed it from common.css